hugegraph-core/src/main/java/com/baidu/hugegraph/auth/StandardAuthManager.java

```diff
@@ -54,13 +61,16 @@
     private final RelationshipManager<HugeBelong> belong;
     private final RelationshipManager<HugeAccess> access;
 
+    private final TokenGenerator tokenGenerator;
+
     public StandardAuthManager(HugeGraphParams graph) {
         E.checkNotNull(graph, "graph");
 
         this.graph = graph;
         this.eventListener = this.listenChanges();
         this.usersCache = this.cache("users");
         this.pwdCache = this.cache("users_pwd");
+        this.tokenCache = this.cache("token");
```

Comment:
should the token expired for 1day by default? (or set to 1h?)

Reply:
1h is too short, no token refresh in current version